tests/test-webapps/test-http2-webapp/src/test/java/org/eclipse/jetty/test/webapp/HTTP2FromWebAppIT.java

```diff
@@ -43,8 +44,10 @@ public void testHTTP2FromWebApp() throws Exception
         Server server = new Server();
 
         SslContextFactory.Server serverTLS = new SslContextFactory.Server();
-        serverTLS.setKeyStorePath("src/test/resources/keystore.p12");
-        serverTLS.setKeyStorePassword("storepwd");
+        serverTLS.setKeyStore(TestKeyStoreFactory.getServerKeyStore());
+        serverTLS.setKeyStorePassword(TestKeyStoreFactory.KEY_STORE_PASSWORD);
+        serverTLS.setTrustStore(TestKeyStoreFactory.getTrustStore());
+        serverTLS.setTrustStorePassword(TestKeyStoreFactory.KEY_STORE_PASSWORD);
```

Comment:
Ditto, another new TrustStore configuration where there wasn't one before.
Remove it.